packages/flutter/test/material/text_field_test.dart

```diff
@@ -250,6 +250,75 @@ void main() {
     await checkCursorToggle();
   });
 
+  testWidgets('cursor has expected defaults', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        overlay(
+          child: const TextField(
+            cursorRadius: Radius.circular(3.0),
```

Comment:
It's useful to test that the TextField's cursor has the correct defaults; so don't set the cursorRadius parameter and then verify that it has its default value. The docs say that the default radius is zero. Perhaps it's actually null and that means zero?

Reply:
Yes, it turns out it is actually null. I have changed the docs and the tests